### ISSUE TITLE
Game crashes on next turn after x number of turns using this mod

### ISSUE THREAD

**Issue by asriel02312** [Sullien/The-Star-Wars#4]
when im playing this mod the game crashes when i get to a certain number of turns on a new game so one game ill get to turn 17 then i go to the next turn and the xcreen goes to black and the game crashes. i tried a new game with natives off it happened on turn 23, i dont know what happened, i downloaded The star wars mod using Unciv and have the latest Unciv downloaded

**Comment by asriel02312** [comment]
i just tried it again whilst playing as sith eternals, if it matters, i was reseearching expansion, Military and research had already been researched and chose conquerors and Weapons depot for policies, and i just made a trade post previous turn,

**Comment by Lux4578r** [comment]
Hi, i have similar issue but the game crashes everytime i am about to obtain a Jedi related unit. Like i only need one turn to obtain a padawan and the game crashed i reloaded but it crashed again. I tried to play with advance development and decided to put the level 4 in configuration so when i only had one turn left to obtain a Jedi Knight the game crashed.

**Comment by Lux4578r** [comment]
I tried again this time under the faction of the Rebel Alliance and the game crashed when i tried to buy a master.

**Comment by asriel02312** [comment]
Yes it keeps crashing after i reload the save and progress with the turn too

**Comment by Sullien** [comment]
> I tried again this time under the faction of the Rebel Alliance and the game crashed when i tried to buy a master.

This should be Fixed now

**Comment by asriel02312** [comment]
> > I tried again this time under the faction of the Rebel Alliance and the game crashed when i tried to buy a master.
> 
> This should be fixed now

Hey Sullien, the latest update it seems has fixed the issue, thank you so much and your mod is amazing, keep up the amazing work!
